### PR TITLE
docs: changelogs overview

### PR DESCRIPTION
**Pull request by vinckr**
Since there was [some consensus that the changelog would not bring much benefit on the website](https://github.com/ory-corp/web/pull/619), here is a small patch that adds a dedicated document with an overview of all our changelogs. 

Is [part of a broader issue](https://github.com/orgs/ory-corp/projects/6/views/10)

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [x] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: docs/ecosystem/changelog.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+id: changelog
+title: Changelogs
+---
+
+> Running security-critical open source technology in a self-service format requires vigilance. We make your life easier by
+> announcing important software updates via the [Ory Security Newsletter](http://eepurl.com/di390P). Never miss an update and
+> **[sign up now to important release updates!](http://eepurl.com/di390P)**
+
+Ory is actively being developed. If you are interested in future development, take a look at the [Roadmap](./roadmap.md). Below
+you find changelogs for Ory services:
```
Comment (tomekpapiernik) [inline docs/ecosystem/changelog.md:11]:
```suggestion
Ory is being actively developed. If you want to know more about the planned features and timelines, read the [Roadmap](./roadmap.md).

Visit these links to get changelogs of individual Ory projects:
```

Comment (tomekpapiernik) [reply docs/ecosystem/changelog.md:11]: > Ory is actively being developed.

I have a problem with that. Is ORY something we develop? Or are Ory PROJECTS or PRODUCTS something we develop? 🤔 Sounds unimportant but I think we should settle on one way of talking about these things and then stick to it.

File: docs/ecosystem/changelog.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- [Ory Kratos Releases](https://github.com/ory/kratos/releases) /
+  [Full CHANGELOG.md](https://github.com/ory/kratos/blob/master/CHANGELOG.md)
+- [Ory Hydra Releases](https://github.com/ory/hydra/releases) /
+  [Full CHANGELOG.md](https://github.com/ory/hydra/blob/master/CHANGELOG.md)
+- [Ory Keto Releases](https://github.com/ory/keto/releases) /
+  [Full CHANGELOG.md](https://github.com/ory/keto/blob/master/CHANGELOG.md)
+- [Ory Oathkeeper Releases](https://github.com/ory/oathkeeper/releases) /
+  [Full CHANGELOG.md](https://github.com/ory/oathkeeper/blob/master/CHANGELOG.md)
+
+Before [upgrading](./upgrading.md) to a newer version, please make sure to check with these documents first.
```
Comment (tomekpapiernik) [inline docs/ecosystem/changelog.md:22]: How about you put it in an admonition and place it BEFORE the changelogs? Could look something like this: 

```
:::warning

Changelogs contain important information about the changes and breaking changes introduced in the code. Please read the relevant project's changelog before you [upgrade](./upgrading.md) to a new version.

:::

File: docs/ecosystem/roadmap.md [hunk 3]
```diff
@@ -71,30 +71,25 @@ All collected money goes directly into open source development.
 
 ### Changelogs
 
-- [Ory Kratos Changelog](https://github.com/ory/kratos/blob/master/CHANGELOG.md) /
-  [Releases](https://github.com/ory/kratos/releases)
-- [Ory Hydra Changelog](https://github.com/ory/hydra/blob/master/CHANGELOG.md) / [Releases](https://github.com/ory/hydra/releases)
-- [Ory Keto Changelog](https://github.com/ory/keto/blob/master/CHANGELOG.md) / [Releases](https://github.com/ory/keto/releases)
-- [Ory Oathkeeper Changelog](https://github.com/ory/oathkeeper/blob/master/CHANGELOG.md) /
-  [Releases](https://github.com/ory/oathkeeper/releases)
+[Visit the Changelog for all Ory Changelogs](./changelog.md)
```
Comment (tomekpapiernik) [inline docs/ecosystem/roadmap.md:74]:
```suggestion
Visit [this page](./changelog.md) to see the changelogs for all Ory projects.
```

WDYT?

File: docs/ecosystem/roadmap.md [hunk 4]
```diff
@@ -71,30 +71,25 @@ All collected money goes directly into open source development.
...
-[Open Issues & Pull Requests](https://github.com/ory/keto/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc) /
-[Issues sorted by popularity](https://github.com/ory/keto/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc)
+[Open Issues & Pull Requests](https://github.com/ory/keto/issues?q=is%3Aopen) /
+[Issues sorted by popularity](https://github.com/ory/keto/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions)
 
 Ory Oathkeeper: [Milestones](https://github.com/ory/oathkeeper/milestones) /
-[Open Issues & Pull Requests](https://github.com/ory/oathkeeper/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
-/ [Issues sorted by popularity](https://github.com/ory/oathkeeper/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc)
+[Open Issues & Pull Requests](https://github.com/ory/oathkeeper/issues?q=is%3Aopen) /
+[Issues sorted by popularity](https://github.com/ory/oathkeeper/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions)
```
Comment (tomekpapiernik) [inline docs/ecosystem/roadmap.md:92]: nit: turn these into unordered lists?

**Review by tomekpapiernik — DISMISSED** [review]

File: docs/ecosystem/changelog.md (hunk 1 above)
Comment (vinckr) [reply docs/ecosystem/changelog.md:11]: Ah you are right. 
But I think in this case "Ory" means the entirety of the "ecosystem" - all open source repos and other products.

File: docs/ecosystem/roadmap.md (hunk 3 above)
Comment (vinckr) [reply docs/ecosystem/roadmap.md:74]: looks much nicer :)

File: docs/ecosystem/roadmap.md (hunk 4 above)
Comment (kevgo) [inline docs/ecosystem/roadmap.md:92]: I suggest merging this into the roadmap chapter on line 14. Milestones are definitely a part of the roadmap. So are issues (where we discuss items that end up on the roadmap) and pull requests (that implement parts of the roadmap). 

I'm not sure it's helpful to see both open issues and pull requests at the same time. These are two very different things, the fact that Github implemented PRs as issues is a quirky implementation detail. I suggest providing separate links to each category.

File: docs/ecosystem/roadmap.md [hunk 5]
```diff
@@ -108,6 +103,6 @@ Ory Oathkeeper: [Milestones](https://github.com/ory/oathkeeper/milestones) /
 
 For all other Ory projects (such as [Ory Dockertest](https://github.com/ory/dockertest/),
 [Ory Fosite](https://github.com/ory/fosite/), [Ory Herodot](https://github.com/ory/herodot/), etc.), please open a discussion in
-`ory/meta`:
+the Ory Org discussion board:
```
Comment (kevgo) [inline docs/ecosystem/roadmap.md:106]: Merge the link below into the text here?

```suggestion
the [Ory Org discussion board](https://github.com/orgs/ory/discussions).
```

**Review by kevgo — COMMENTED** [review]
Thanks for adding this to the docs!

**Comment by tomekpapiernik** [comment]
Oooopsie!
```
Error: Docs markdown link couldn't be resolved: (../cli/ory-identities-list.md) in /home/runner/work/docs/docs/docs/concepts/personal-access-token.mdx for version current
```

**Review by tomekpapiernik — APPROVED** [review]

**Pull request merged**